frontend-react/src/components/FileHandlers/FileHandler.tsx

```diff
@@ -222,49 +261,62 @@ function FileHandler() {
                 },
             });
         }
+        const isFileSuccess =
+            eventData?.overallStatus === OverallStatus.VALID &&
+            !(eventData?.errorCount || eventData?.warningCount);
+        if (isFileSuccess) {
+            handleNextFileHandlerStep(3);
+        } else {
+            handleNextFileHandlerStep();
```

Comment:
Small/subjective point, but it feels very arbitrary to call this with no argument.